networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -82,6 +82,9 @@ class ConfigGenerator:
     }
     ignore_tags = {"cc-net-driver-ignore"}
     extra_vlan_tag = "cc-net-driver-extra-vlan"
+    force_speed_tag = 'cc-net-driver-force-speed'
+    speed_tag_regex = re.compile(r'^cnd-net-portspeed-(?P<speed>\d+g)$')
```

Comment:
All other regexes end with `_re` (see above). Do you want to do that here as well?

Reply:
Agreed.